www/pages/register.js

```diff
@@ -117,27 +118,27 @@ const Register = (props) => {
           name="email"
           label="email"
           variant="standard"
-          value={fields.email}
-          onChange={handleChange}
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
           className={styles.text_field}
-          error={emailError}
+          helperText={emailError}
         />
         <TextField
           name="username"
           label="username"
           variant="standard"
-          value={fields.username}
-          onChange={handleChange}
+          value={username}
+          onChange={(e) => setUsername(e.target.value)}
           className={styles.text_field}
-          error={usernameError}
+          helperText={usernameError}
```

Comment:
I think this got messed up when we switched to mui. But the error messages are working once again.